### DISCUSSION TITLE
Share apps to nearby devices over LAN

### DISCUSSION THREAD

**Discussion by trymeouteh** [pacstall/pacstall#870] [Ideas]
F-Droid which is an app store client allow you to install apps on Android from the F-Droid repository or any other F-Droid repo which anyone can self host a F-Droid repo also has a feature called "Nearby" which allows you to share installed apps to other devices nearby over LAN.

It is a cool feature and if the internet goes down for whatever reason and one of your friends has an app on their computer or you have an app installed on another computer in your home, you can easily install the app from the other devices over the LAN.

Something to consider adding to the Pacstall GUI.

**Comment by Elsie19** [comment]
So you want to be able to share a finished Deb to others?

**Reply by trymeouteh** [reply]
Any Pacstall app.

**Reply by Elsie19** [reply]
Some apps need internet to build. How would that get transferred to another device?

**Reply by wizard-28** [reply]
I think he's talking about the pre-built debs using `-B` flag.

**Reply by Elsie19** [reply]
> So you want to be able to share a finished Deb to others?

That's what I asked here.

**Reply by trymeouteh** [reply]
I did not know some packages required the internet to build them. So in that case, I guess it would be for packages that do not require the internet such as pre-built debs.
